pagerenderer/vue/core/src/main/java/com/peregrine/pagerender/vue/models/PageModel.java

```diff
@@ -320,7 +320,13 @@ public Tag(Resource r) {
             this.path = r.getPath();
             this.path = path.substring(path.indexOf("/jcr:content"));
             this.name = r.getName();
```

Comment:
looks like this is a former error. the name of the tag should be fetched from the tag itself (same place as the value) and not from the name of the node that references it (probably fails to render the tag name correctly if the name of the tag ever changes)

Reply:
That is correct, got objects working when deleted, can't use/see them. One other problem, cannot save/edit tags once placed on page. Think it has to do with our tags.infinity.json and how the data is stored, the tag objects are still there after deleting. 
<img width="1324" alt="Screen Shot 2020-05-15 at 1 23 53 PM" src="https://user-images.githubusercontent.com/49845255/82093106-548a1480-96af-11ea-8d84-7cd78e24706d.png">